src/components/DataTable/DataTable.tsx

```diff
@@ -472,7 +472,12 @@ const DataTable = (props: Props) => {
   return (
     <ThemeProvider theme={theme}>
       <div data-test='datatable' className={tableClasses}>
-        <div className={'row'}>
+        <div style={{
+          display: "flex",
+          flexWrap: "wrap",
+          justifyContent: "space-between",
```

Comment:
Is there any reason why we you chose inline styling here?

Reply:
No specific reason. I will make a change to this.